models/User.js

```diff
@@ -1,5 +1,102 @@
 const mongoose = require("mongoose");
 const Schema = mongoose.Schema;
+
+//Do we want start dates required?
+
+//Create education schema for use in ResumeSchema
+const EducationSchema = new Schema({
+  name:{
+    type: String,
+    required: true,
+  },
+  program:{
+    type: String,
+    required: true,
+  },
+  grade:{
+    type: Number,
+  },
+  start_date:{
+    type: Date,
+  },
+  end_date:{
+    type: Date,
+  }
+});
+
+//paid/volunteer experience schema for use in ResumeSchema
+const ExperienceSchema = new Schema({
+  title:{
+    type: String,
+    required: true,
+  },
+  company:{
+    type: String,
+    required: true,
+  },
+  start_date:{
+    type: Date,
+  },
+  end_date:{
+    type: Date,
+  },
+  description:{
+    type: String
+  }
+});
+
+//Project Schema for use in ResumeSchema
+const ProjectSchema = new Schema({
+  name:{
+    type: String,
+    required: true,
+  },
+  stack:{
+    type: String,
+  },
+  description:{
+    type: String,
+  },
+  links:[String]
```

Comment:
```suggestion
  links:[{text: String, uri: String}]
```
This way we could have a link like "[my github!](https://github.com)", or even just hide the scheme.

Reply:
I don't think that this is a good change because if these resumes are something that will be easily ported to pdfs, and handed out on paper, that will be very counter-intuitive. Maybe I am missing the scope of the project though and these will be web resumes.

Reply:
I was under the impression most people would be exporting these as web resumes for 2 big reasons: 
1. Most job applications are done digitally anyway, especially in our case (young CS professionals).
2. Nobody is handing out paper resumes in a pandemic, and likely won't be for a while

That said, maybe someone will want a prinout in which case the user can set `text = ""` or something to that effect and we will handle that when we validate user input.